components/rhs_root_post/__snapshots__/rhs_root_post.test.jsx.snap

```diff
@@ -855,52 +773,6 @@ exports[`components/RhsRootPost should match snapshot when flagged 1`] = `
             postId="id"
           />
         </div>
-        <div
```

Comment:
Should we have any test that its snapshot includes this piece of HTML?

Reply:
✅  yes included a test